src/pages/item/ItemInfo.jsx

```diff
@@ -73,18 +74,20 @@ const ItemInfo = () => {
 
   const addToCardBtnHandler = () => {
     if (countProduct > 0 && !isInCard(product.id)) {
-      addingProduct(product);
+      addingProduct(product, countProduct);
     }
+    colorIsBlack();
+    productCountReSet(0);
   };
   //component
 
   return (
     <main className="item-page">
       <header className="header-of-item">
-        <Link key="1" to="/listofproducts">
+        <Link key="1" to="/listofproducts" onClick={colorIsBlack}>
           <FiChevronLeft />
         </Link>
-        <Link key="2" to="/card">
+        <Link key="2" to="/card" onClick={colorIsBlack}>
```

Comment:
change function name to `changeColorToBlack` in `useColorChange` and also update other color changer function names